### DISCUSSION TITLE
First web admin login

### DISCUSSION THREAD

**Discussion by ghost** [Syslifters/sysreptor#2] [Q&A] [answered]
Hi ! 

I wanted to try your work as I like to keep in touch with security tools.
The installation can't be easier fo sure. 
I might have missed it but I can't see any informations about the first login. 
I have look for it in documentation, Dockerfile and .env but I have probably miss the information somewhere.

Could you, please, point me where I can fount it ?

Best regards.

**Comment by theguly** [comment, marked as answer]
I believe you should:
docker compose exec app python3 manage.py createsuperuser
or
docker compose exec app python3 manage.py help changepassword

because logins are related to django app and as far as i remember there is no default account (for obvious security reasons :) )

**Reply by ghost** [reply]
That's it !
Thanks a lot.
